modules/apps/digital-signature/digital-signature-test/src/testFunctional/macros/DigitalSignature.macro

```diff
@@ -21,6 +21,41 @@ definition {
 			locator1 = "Header#H1_TITLE");
 	}
 
+	macro canDataPersistAfterEnabled {
```

Comment:
Could you change this macro name to: `canSeeDataPersistAfterEnabled` and add this code:
```
var digitalSignatureApiUsername = PropsUtil.get("digital.signature.api.username");
		var digitalSignatureApiAccountId = PropsUtil.get("digital.signature.api.accountId");
		var digitalSignatureAccountBaseUri = PropsUtil.get("digital.signature.account.base.uri");
		var digitalSignatureIntegrationKey = PropsUtil.get("digital.signature.integration.key");
		var digitalSignatureRsaPrivateKey = PropsUtil.get("digital.signature.rsa.private.key");
```
due to it is needed to asset these values